examples/contrib/export_har/test_exporthar.py

```diff
@@ -0,0 +1,46 @@
+import json
+from pathlib import Path
+
+import pytest
+from exporthar import ExportHar
+
+from mitmproxy import io
+from mitmproxy import types
+
+here = Path(__file__).parent.absolute()
+
+
+@pytest.mark.parametrize(
+    "log_file", [pytest.param(here / "flows/logfile", id="logfile")]
+)
+def test_errors(log_file):
+    e = ExportHar()
+    path = open(log_file, "rb")
+    flows = io.FlowReader(path).stream()
+    pytest.raises(
+        FileNotFoundError,
+        e.export_har,
+        flows,
+        types.Path("unknown_dir/testing_flow.har"),
+    )
+
+
+@pytest.mark.parametrize(
+    "log_file", [pytest.param(here / "flows/logfile", id="logfile")]
+)
+def test_exporthar(log_file, tmp_path):
+    e = ExportHar()
+    path = open(log_file, "rb")
+    flows = io.FlowReader(path).stream()
+    e.export_har(flows, types.Path(tmp_path / "testing_flow.har"))
+    correct_har = json.load(open("flows/correct_flows.har"))
+    testing_har = json.load(open(tmp_path / "testing_flow.har"))
```

Comment:
Let's not use `open` without a context manager (`with open(...): ...`) . That leaves open file handles around, which will break on Windows. You can run `.read_bytes()` on path objects and then just `json.load`.